demo_apps/Azure_API_example/azure_api_example.ipynb

```diff
@@ -0,0 +1,610 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use Azure API with Llama 2\n",
+    "\n",
+    "This notebook shows examples of how to use Llama 2 APIs offered by Microsoft Azure. We will cover:  \n",
+    "* HTTP requests API usage for Llama 2 70B pretrained and chat models in CLI\n",
+    "* HTTP requests API usage for Llama 2 70B pretrained and chat models in Python\n",
+    "* Plug the APIs into LangChain\n",
+    "* Wire the model with Gradio to build a simple chatbot with memory\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisite\n",
+    "\n",
+    "Before we start building with Azure Llama 2 APIs, there are certain steps we need to take to deploy the models:\n",
+    "\n",
+    "* Register for a valid Azure account with subscription \n",
+    "* Make sure you have access to [Azure AI Studio](https://learn.microsoft.com/en-us/azure/ai-studio/what-is-ai-studio?tabs=home)\n",
+    "* Create a project and resource group\n",
+    "* Select Llama models from Model catalog\n",
```

Comment:
My take here is that user might already have an available resource because resource can be created in other places and shared across different Azure products. It is hard to cover everything entry point here.  I added specific Llama 2 deployment tutorial there and article to describe resource.